mlrun/api/db/sqldb/db.py

```diff
@@ -1231,20 +1321,40 @@ def _add_utc_timezone(obj, attribute_name):
     def _transform_feature_set_model_to_schema(
         feature_set_record: FeatureSet, tag=None,
     ) -> schemas.FeatureSet:
+        def _transform_label_list_to_dict(label_list):
+            return {label.name: label.value for label in label_list}
```

Comment:
Sounds like something that will be useful for other resources as well, you can move to outer scope

Reply:
ok. I moved it to the helpers.py file.